### ISSUE TITLE
0002420:
Inform users about new placed task

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#1290]
**Reported by maitre on 19 Mar 2010 22:12**

**Version:** Mialena (2010-03) Milestone 1

If User A places a new task for user B, Bh still won´t be informed about this new task. 

Proposal: Giving the possibility to sent user B a automatic knowledge email 

**Additional information:** see http://www.tine20.org/forum/viewtopic.php?f=12&amp;t=320&amp;p=12490#p1140

**Comment by pschuele** [comment]
duplicate of #978

**Comment by Gloirin** [comment]
**Comment posted by llldannylll on 9 Oct 2013 12:59**

Any news on this?
Its a very useful feature.

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 9 Oct 2013 13:09**

we currently have no plans for this. maybe this will be on the roadmap for the release after Collin.

**Comment by Gloirin** [comment]
**Comment posted by patrickk_gn on 23 May 2014 18:28**

I wait on this very important feature too. Is it possible, that you implement it within the next release? Thanks!
